src/connection.js

```diff
@@ -1723,6 +1723,9 @@ Connection.prototype.STATE = {
       this.sendInitialSql();
     },
     events: {
+      socketError: function socketError() {
```

Comment:
what about a test?

Reply:
Hmm, I don't see any helpful existing tests that check this transition :/

Reply:
Right :)